### DISCUSSION TITLE
Can't close an issue on one of my repositories

### DISCUSSION THREAD

**Discussion by DeveloppeurPascal** [community/community#27576] [Projects and Issues] [answered]
Hi

I'm trying to close this issue https://github.com/DeveloppeurPascal/TetrisLike/issues/3 but I can't.

The page as me to add a comment and when I comment and try to close, it only adds the comment and don't close the issue.

Do you have any similar problem ?

**Comment by jfullstackdev** [comment]
I just tested it by now 

![image](https://user-images.githubusercontent.com/47092464/181742171-3be0d691-8c39-484e-a627-913987da633d.png)

**Reply by DeveloppeurPascal** [reply]
thanks, so it's not a problem on my account, it's more global

**Reply by jfullstackdev** [reply]
I can close it, forgot to mention that

**Reply by DeveloppeurPascal** [reply]
ok, so I'm alone :(

when I try to close I have the error "You can't comment at this time — your comment cannot be blank."
when I "close with comment", the issue stay opened and the comment is added.

**Reply by jfullstackdev** [reply]
can you have a screenshot of that, full screen ?

**Comment by DeveloppeurPascal** [comment]
![Capture d’écran 2022-07-29 à 20 57 55](https://user-images.githubusercontent.com/29439699/181826729-fe441f62-83a9-4a4f-9913-1e001c3ea2f1.png)

@github has probably changed something this week that doesn't work anymore on Safari or there are network problems today because some files doesn't load properly depending on the page where I am on GitHub.

**Reply by ymaxee** [reply]
You have to comment something inside, then you can close it

**Reply by jfullstackdev** [reply]
good you have the screenshot, see the error at Developer Tools ? different browsers have different errors sometimes, but I'm using Chrome for GitHub and most of the time this browser is okay,

what you can try is log in in Incognito ( private browsing ) mode and see whether it will work

**Comment by DeveloppeurPascal** [comment, marked as answer]
Today I can't add labels to repositories and when I click on "view more" in a discussion a part of the page disappear. GitHub join the list of "professional" website for developers to be broken for Safari 13. :(

I'll try with Chrome.

**Comment by DeveloppeurPascal** [comment]
and of course, with Google Chrome it's ok...

frontend developers seriously annoy me ! there are no reason for this forced deprecation.

**Comment by laurencelundblade** [comment]
Broken in Safari 13 for me. Works in Firefox.
